src/Landing.js

```diff
@@ -4,24 +4,27 @@ import "./css/form.css";
 
 const Landing = () => {
   return (
-    <div class="main">
-      <div class="main-title">What is Mustang Messenger?</div>
-      <div class="main-body">
-        Mustang Messenger is a platform where you can connect and message with
-        other users in private 1-on-1. Sign up below with an email to get
-        started today!
-      </div>
-      <div class="email-form">
-        Enter your email to get started:
-        <form id="signup-email">
-          <input
-            id="enter-signup-email"
-            type="text"
-            placeholder="Enter email here..."
-          />
-          <input id="go-button-signup-email" type="button" value="Go" />
-        </form>
-        Already have an account? Sign up <Link to="/signup">here.</Link>
+    <div className="main">
+      <div className="email-form">
+        <div className="main-title">
+          What is Mustang Messenger?
+        </div>
+        <br />
+        <div className="main-body">
+          Mustang Messenger is a platform where you can connect and message with other users in private 1-on-1.
+          This application features unique user accounts and the ability to message other users.
+          All you need is an email and password, and you, too can send and receive messages with other users of the platform!
+          <br/><br/>
+          Sign up with an email to get started today!
```

Comment:
Can we at least have a static button or link here.

Reply:
Added in most recent commit, feel free to re-reivew.